en/nearest-neighbor-search.md

```diff
@@ -198,8 +204,16 @@ Vespa supports six different [distance-metrics](reference/schema-reference.html#
 * `hamming`
 * `geodegrees`
 
-### Configure rank profiles for nearest neighbor search
+The distance-metric is a property of the field.
+This is obvious when `index` is set on the vector field,
+as the distance metric is used when building the index structure.
+Without `index`, no extra data structures are built for the field,
+and the distance-metric setting is used when calculating the distance at query-time.
+It still makes sense to have the metric as a field-property,
+as the field values are often produced using a specific distance metric.
+Finally, without `index` there is no need to restart Vespa after redeploying with a new distance metric.
```

Comment:
I am not sure if this is true, @geirst (the restart required, if there is no HNSW index)

Reply:
Actually, https://docs.vespa.ai/en/reference/schema-reference.html#changes-that-require-restart-but-not-re-feed says a restart is required regardless of an index, will remove the last sentence.

Reply:
Yes, changing the `distance-metric` triggers a restart action from the config model.